spring-cloud-starter-stream-processor-tensorflow/README.adoc

```diff
@@ -1,11 +1,32 @@
 //tag::ref-doc[]
 :image-root: https://raw.githubusercontent.com/spring-cloud-stream-app-starters/tensorflow/master/images
+:master-root: https://raw.githubusercontent.com/spring-cloud-stream-app-starters/tensorflow/master
 = TensorFlow Processor
 
 A processor that evaluates a machine learning model stored in TensorFlow Protobuf format.
 
+Following snippet shows how to export a `TensorFlow` model into `ProtocolBuffer` binary format as required by the Processor.
+```python
+from tensorflow.python.framework.graph_util import convert_variables_to_constants
+...
+SAVE_DIR = os.path.abspath(os.path.curdir)
+minimal_graph = convert_variables_to_constants(sess, sess.graph_def, ['<model output>'])
+tf.train.write_graph(minimal_graph, SAVE_DIR, 'my_graph.proto', as_text=False)
+tf.train.write_graph(minimal_graph, SAVE_DIR, 'my.txt', as_text=True)
+```
+
 image::{image-root}/TensorFlowProcessorArcutectureOverview.png[]
 
+The `--tensorflow.model` property configures the Processor with the location of the serialized Tensorflow model.
+
+The `TensorflowInputConverter` implementations converts the input data into the format, specific for the given model.
```

Comment:
I think to align with the next sentence the `implementations` word is redundant.

Reply:
+1